### DISCUSSION TITLE
About K-Space Reconstruction

### DISCUSSION THREAD

**Discussion by jinfeng2411** [facebookresearch/fastMRI#222] [Q&A]
In paper [**Joint Deep Model-based MR Image and Coil Sensitivity Reconstruction Network
(Joint-ICNet) for Fast MRI**]  and [**GrappaNet: Combining Parallel Imaging with Deep Learning for Multi-Coil
MRI Reconstruction**], they used U-Net to reconstruct parallel MRI in K-space, and achieved a good effect. But in my experiments, using U-net in K-space, the results are terrible. Can anyone give me some advice? I would be very grateful for your help, thanks!

**Comment by mmuckley** [comment]
The [DIRECT repository](https://github.com/NKI-AI/direct) has an implementation of Joint-ICNet that you could look into, as well as KIKI-net. The [fastmri-reproducible-benchmark repository](https://github.com/zaccharieramzi/fastmri-reproducible-benchmark) repository also has an implementation of KIKI-net. I don't now of any open-source implementations of GRAPPANet.

CC @anuroopsriram
